src/languages/actionscript.js

```diff
@@ -92,20 +96,16 @@ export default function(hljs) {
       hljs.C_BLOCK_COMMENT_MODE,
       hljs.C_NUMBER_MODE,
       {
-        className: 'class',
-        beginKeywords: 'package',
-        end: /\{/,
-        contains: [ hljs.TITLE_MODE ]
+        beforeMatch: /\b(package)\s+/,
```

Comment:
this is super slick!

Reply:
It may still be replaced by multi-match though since that can do the same thing... not sure.